core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -10543,26 +10543,19 @@ RelExpr *Update::bindNode(BindWA *bindWA)
 	{
 	  xnsfrmHbaseUpdate = TRUE;
 	}
-      else if ((CmpCommon::getDefault(HBASE_TRANSFORM_UPDATE_TO_DELETE_INSERT) == DF_SYSTEM) &&
-	       (getTableDesc()->getNATable()->hasSecondaryIndexes()))
-	{
-	  xnsfrmHbaseUpdate = TRUE;
-	}
-      else if (avoidHalloween())
-	{
-	  xnsfrmHbaseUpdate = TRUE;
-	}
       else if (getCheckConstraints().entries())
-	{
-	  xnsfrmHbaseUpdate = TRUE;
-	}
+       {
```

Comment:
I suppose the default attribute can be removed from nadefaults.cpp and DefaultConstants.h also?